library/src/main/java/com/schibsted/spain/barista/interaction/BaristaChipInteractions.kt

```diff
@@ -0,0 +1,13 @@
+package com.schibsted.spain.barista.interaction
+
+import com.schibsted.spain.barista.internal.performAction
+import com.schibsted.spain.barista.internal.util.resourceMatcher
+import com.schibsted.spain.barista.internal.viewaction.ChipViewActions.removeChip
+
+object BaristaChipInteractions {
+
+  @JvmStatic
+  fun closeChip(resId: Int) {
```

Comment:
I understand that you name it `close` instead of `remove` to avoid conflicting with the action itself...